docs/quickstart/cli.md

```diff
@@ -0,0 +1,293 @@
+# CLI
+
+This page works through the SQLMesh example project using the SQLMesh command-line interface.
+
+## 1. Create the SQLMesh project
+First, create a project directory and navigate to it:
+
+```bash
+mkdir sqlmesh-example
+```
+```bash
+cd sqlmesh-example
+```
+
+If using a python virtual environment, ensure it's activated first by running the `source .env/bin/activate` command from the folder used during [installation](../installation.md).
+
+Create a SQLMesh scaffold with the following command:
+
+```bash
+sqlmesh init
+```
+
+See the [quick start overview](../quick_start.md#project-directories-and-files) for more information about the project directories, files, data, and models.
+
+## 2. Plan and apply environments
+### 2.1 Create a prod environment
+
+SQLMesh's key actions are creating and applying *plans* to *environments*. At this point, the only environment is the empty `prod` environment.
+
+The first SQLMesh plan must execute every model to populate the production environment. Running `sqlmesh plan` will generate the plan and the following output: 
+
+```bash linenums="1"
+$ sqlmesh plan
+======================================================================
+Successfully Ran 1 tests against duckdb
```

Comment:
Nit: maybe we can clean this up a bit: "Successfully ran 1 test(s) against DuckDB".

(Note that this change would need to be cascaded in several places in the repo -- happy to assist with this!)

Reply:
Feel free to ignore this comment if it's too much of a hassle. I just noticed all screenshots contain this message as well, so it might be more annoying than I initially expected 😅 .

Reply:
Yeah, I've had enough screenshot time for the moment 😅